L1Trigger/RPCTechnicalTrigger/src/RBCProcessRPCSimDigis.cc

```diff
@@ -344,7 +267,7 @@ void RBCProcessRPCSimDigis::setDigiAt( int sector, int digipos )
 void RBCProcessRPCSimDigis::setInputBit( std::bitset<15> & signals , int digipos ) 
 {
   
-  int bitpos = m_layermap[digipos];
+  int bitpos = s_layermap.at(digipos);
```

Comment:
This is a functional difference, the code will now throw if `digipos` is not in the container.